tests/conftest.py

```diff
@@ -95,6 +95,7 @@
 from tornado import netutil
 from tornado.platform.asyncio import AnyThreadEventLoopPolicy
 
+import _pytest.config
```

Comment:
this import is not correct I think.

Reply:
Right, I wanted to ask about this. It is documented to be there though, `pytest.config` does not exist. Any suggestions?

Reply:
https://github.com/pytest-dev/pytest/blob/main/src/pytest/__init__.py

Reply:
I ended up typing it in quotes: it runs fine and mypy complains now but will be happy once it gets released.